### DISCUSSION TITLE
Support CheckList in mdxeditor

### DISCUSSION THREAD

**Discussion by noteverseio** [mdx-editor/editor#154] [Ideas]
Hey dear mdxeditor author,

Is it possible to support check list by typing '[]' or '[x]'?

Most of the markdown e.g GFM, supports check and unchecked list if use inputs '[]' or '[x]', 

'''
[x] read paper today
[] walk the dog
[] watch video
'''

**Comment by petyosi** [comment]
Yes, it is possible. PRs on the matter are welcome, there's an open issue for that.

**Comment by noteverseio** [comment]
I was able to implement a checklist based on lexical checklists and the mdast extension, will clean up a few things and post a PR once it's done ;)

**Reply by forhappy** [reply]
PR https://github.com/mdx-editor/editor/pull/159 implemented the check list feature ;)

**Comment by petyosi** [comment]
Merged and released. Thank you.
